### PR TITLE
Formatted Section 6.5 (Grin Design Choice Criticisms...) for review

### PR DESCRIPTION
**Pull request by anselld**
Hi @neonknight64, I've completed the formatting of Section 6.5 in accordance with the TLU Style Guide. Before I continue with the next stage (technical editing), please could you let me know if you're happy with the formatting.
Thanks and regards
Denise

### CODE REVIEW

**Comment by anselld** [comment]
Hi @neonknight64, when checking the links, I could not access [10], [27] and [34].

File: src/protocols/grin-design-choice-criticisms/MainReport.md [hunk 1]
```diff
@@ -36,265 +52,383 @@ Their primary motivations for selecting a static emission rate are:
...
 
-[[10]] Debtcoin: Credit, debt, and cryptocurrencies, Landon Mutch, https://cryptoinsider.21mil.com/debtcoin-credit-debt-and-cryptocurrencies/, Date accessed: 2018-11-06.
+[[10]] L. Mutch, "Debtcoin: Credit, Debt, and Cryptocurrencies" [online]. 
+Available: <https://cryptoinsider.21mil.com/debtcoin-credit-debt-and-cryptocurrencies/>. Date accessed: 2018&#8209;11&#8209;06.
 
 [10]: https://cryptoinsider.21mil.com/debtcoin-credit-debt-and-cryptocurrencies/
-"Debtcoin: Credit, debt, and
-cryptocurrencies, Landon Mutch"
+"Debtcoin: Credit, Debt, and
+Cryptocurrencies, Landon Mutch"
```
Comment (neonknight64) [inline src/protocols/grin-design-choice-criticisms/MainReport.md]: This link is still available on the "wayback machine" even if the website has disappeared. The specified access date still makes this a valid reference.

File: src/protocols/grin-design-choice-criticisms/MainReport.md [hunk 2]
```diff
@@ -36,265 +52,383 @@ Their primary motivations for selecting a static emission rate are:
...
+for Elliptic-curve Cryptography"
 
-[[26]] RocksDB, https://rocksdb.org/, Date accessed: 2018-10-10.
+[[26]] RocksDB [online]. Available: <https://rocksdb.org/>. Date accessed: 2018&#8209;10&#8209;10.
 
 [26]: https://rocksdb.org/
 "RocksDB"
 
-[[27]] LevelDB, http://leveldb.org/, Date accessed: 2018-10-15.
+[[27]] LevelDB [online]. Available: <http://leveldb.org/>. Date accessed: 2018&#8209;10&#8209;15.
```
Comment (neonknight64) [inline src/protocols/grin-design-choice-criticisms/MainReport.md]: The new link can be used for line 372 and 374:
https://github.com/google/leveldb

File: src/protocols/grin-design-choice-criticisms/MainReport.md [hunk 3]
```diff
@@ -36,265 +52,383 @@ Their primary motivations for selecting a static emission rate are:
...
 
-[[33]] HyperDex Benchmark, Symas Corp, http://www.lmdb.tech/bench/hyperdex/, Date accessed: 2018-10-29.
+[[33]] HyperDex Benchmark, Symas Corp [online]. Available: <http://www.lmdb.tech/bench/hyperdex/>. 
+Date accessed: 2018&#8209;10&#8209;29.
 
 [33]: http://www.lmdb.tech/bench/hyperdex/
 "HyperDex Benchmark, Symas Corp"
 
-[[34]] Grin - Basic Wallet, https://github.com/mimblewimble/grin/blob/master/doc/wallet/usage.md, Date accessed: 2018-10-15.
+[[34]] Grin - Basic Wallet [online]. Available: <https://github.com/mimblewimble/grin/blob/master/doc/wallet/usage.md>.
```
Comment (neonknight64) [inline src/protocols/grin-design-choice-criticisms/MainReport.md]: Not sure how to fix the link, file does not exist anymore. Easiest is to remove this reference.

Comment (anselld) [reply src/protocols/grin-design-choice-criticisms/MainReport.md]: Thanks.

File: src/protocols/grin-design-choice-criticisms/MainReport.md (hunk 1 above)
Comment (anselld) [reply src/protocols/grin-design-choice-criticisms/MainReport.md]: Thanks for your help.

File: src/protocols/grin-design-choice-criticisms/MainReport.md (hunk 2 above)
Comment (anselld) [reply src/protocols/grin-design-choice-criticisms/MainReport.md]: Hi @neonknight64, thanks, will do. I'll let you know when I've completed the editing of the section.

**Comment by anselld** [comment]
Hi @neonknight64 , thanks.

**Comment by anselld** [comment]
Hi @hansieodendaal, for info.

**Pull request merged**